.htmltest.yml

```diff
@@ -3,3 +3,5 @@ IgnoreDirectoryMissingTrailingSlash: true
 CheckExternal: false
 IgnoreAltMissing: true
 IgnoreEmptyHref: true
+IgnoreInternalURLs:
```

Comment:
May I ask why you've added this?

Reply:
Sure, because it was failing if not. If you look at the comments before, @JorTurFer suggested me to do it, as in current keda release this scaler is not present.

Reply:
@tomkerkhove , this is because we have some links to latest version on each scaler page. As new scalers aren't in latest version (because they are unreleased yet) the broken link check fails. Once we release the new version, we have to remove these ingores. 
I'm not sure if we can prevent this somehow, but currently that's the best option I have found. Maybe we should also update the release process to add this step of removing the ignores